src/index.ts

```diff
@@ -335,20 +349,14 @@ const createTab = async ({
       const mappedColumn = sourceLocation.column + 1;
       const mappedLine = sourceLocation.line;
       const mappedPath = sourceLocation.source || url.pathname;
-      // If the stack frame has a name (i.e. function name), then display it
-      // _unless_ the stack frame is the first frame
-      // because if the function name is displayed in the first stack frame,
-      // then Jest cannot parse the stack trace to display the code frame
-      const location =
-        stackItem.name && !isFirst // Have to use isFirst instead of array loop index because function has early returns
-          ? `${stackItem.name} (${mappedPath}:${mappedLine}:${mappedColumn})`
-          : `${
-              // The first line has to be an absolute path,
-              // otherwise Jest won't recognize it for the code frame
-              isFirst ? path.join(process.cwd(), mappedPath) : mappedPath
-            }:${mappedLine}:${mappedColumn}`;
-      isFirst = false;
-      return `    at ${location}`;
+      const stackLine = printStackLine(
+        // For the code frames, Jest will only recognize absolute paths
+        path.join(process.cwd(), mappedPath),
+        mappedLine,
+        mappedColumn,
+        stackItem.name,
+      );
+      return stackLine;
```

Comment:
Nit: do we need to define this and return it on separate lines?

GitHub's suggestion tool isn't working for me, but could we just do `return printStackLine(...)`?

Reply:
Yeah I used to have a line in between but it is gone now, good catch